models.py

```diff
@@ -9,9 +9,9 @@ def save(self):
         db.session.add(self)
         try:
             db.session.commit()
-        except Exception:
+        except Exception as err:
             db.session.rollback()
-            raise Exception
+            raise err
```

Comment:
@brittanylau I figured it might be good to re-raise the original error after rescuing it so we can easily understand what went wrong. Let me know if we shouldn't do this for some reason!

Reply:
Sounds good! I think `handle_server_error` in `app.py` still catches this, so the client doesn't see a funky error response 😄